fog/ledger/server/src/db_fetcher.rs

```diff
@@ -133,25 +139,51 @@ impl<DB: Ledger, E: LedgerEnclaveProxy + Clone + Send + Sync + 'static> DbFetche
                 log::info!(self.logger, "Db fetcher thread stop requested.");
```

Comment:
If we request the thread to be stopped, then should we also set the server to unready? I would guess that this isn't necessary since we normally stop this thread when we call `stop`, which seems to only be called when the ledger server that encapsulates the `db_fetcher` is dropped. 

However, I just want to make sure that we wouldn't expect Kubernetes to display a ready status for a server that is being / has just been dropped?

Reply:
I think the server is probably going to drop before K8s has a chance to react to the signal, because k8s works by periodically polling the health end point, and not that frequently